src/transaction/log_manager.h

```diff
@@ -246,4 +246,12 @@ void LOG_CS_PROMOTE (THREAD_ENTRY * thread_p);
 bool LOG_CS_OWN (THREAD_ENTRY * thread_p);
 bool LOG_CS_OWN_WRITE_MODE (THREAD_ENTRY * thread_p);
 
+// *INDENT-OFF*
+/* Interface used by log flush to notify threads. */
+class log_flush_lsa
```

Comment:
When design is clear, we might consider using bound functions as alternative to virtual classes for notifying GCM's when stream_pos/lsa have been "acknowledged".

Reply:
Only, if bring a benefit - more clear, faster execution.